src/app/profile/[id]/page.tsx

```diff
@@ -0,0 +1,116 @@
+"use client";
```

Comment:
How come we need "use client" here? We should only add this sparingly.

Reply:
I used a shadcn table component and it said it needed it. Should I just switch to a regular table?

Reply:
Ah I see. Best practice is to try and reduce the usage of "use client" and move it as far down the component tree as possible. This is the top level component so the whole page would be impacted. See https://nextjs.org/docs/app/building-your-application/rendering/composition-patterns to understand more.